### PR TITLE
template(structure): <create node/express app>

### PR DESCRIPTION
**Pull request by VernL**
Create app with express generator
Intergrate index.html & map.html views
Closes issue #44

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
-node_modules/
\ No newline at end of file
+node_modules/
+.idea
```
Comment (RichardLitt) [inline .gitignore]: What's .idea?

File: package.json [hunk 2]
```diff
@@ -1,10 +1,10 @@
 {
   "name": "the-travel-shelf",
-  "version": "1.0.0",
+  "version": "0.0.1",
```
Comment (RichardLitt) [inline package.json]: Bad habit to downgrade, and 1.0.0 is the new default start. change to 1.0.1?

File: .gitignore (hunk 1 above)
Comment (VernL) [reply .gitignore]: It's a project folder generated by the WebStorm IDE.

Comment (RichardLitt) [reply .gitignore]: Cool. Add this to a global .gitignore, maybe? Bad habit to add this because it adds code that is only relevant to you.

**Pull request closed without merging**